### ISSUE TITLE
An exception occurred (ARM11)

### ISSUE THREAD

**Issue by TakumaEN** [LumaTeam/Luma3DS#2080]
**System model:**
New 3DS

**SysNAND version:**
11.17.0-50U SysNAND

**Entrypoint:**
Booted from SD via B9S

**Luma3DS version:**
v13.1.2

**Luma3DS configuration/options:**

Screen brightness: ( 1 )

Splash: ( Off )

PIN lock: ( Off )

New 3DS CPU: ( Off )

Hbmenu autoboot: ( Off )

Enable loading external FIRMs and modules: ( x )

Enable game patching: ( x )

Redirect app. syscore threads to core2: ( )

Show NAND or user string in System Settings: ( x )

Show GBA boot screen in patched AGB_FIRM: ( )

**Explanation of the issue:**
My 3DS crashes at random while playing Persona Q2. By the error screen, it says:

> Processor: Arm 11 (Core 0)
> Exception type: data abort
> Fault status: Translation - Section
> Current process: AQ2 (00040000001D7100) 

Not sure if the issue is game related, so I've attempted to see if this happens while playing other games in my system, but so far, not, but that's not to say it couldn't be. This is proving to be quite difficult to troubleshoot, because it really is at random. Sometimes it crashes a few minutes in, and sometimes over the span of 30 minutes or so, it's very unpredictable!

[UPDATE]
Indeed it only happens with Persona Q2. Tried reinstalling the game to make sure it downloaded correctly and the problem persists.

I've left the log in the attachment just in case.

[crash_dump_00000000.zip](https://github.com/user-attachments/files/17002624/crash_dump_00000000.zip)

[UPDATE 2]
As of January 2nd, 2025, ran another attempt using the luma v.13.3 and still did not fix the issue. I'll leave the updated log below. 
[crash_dump_00000003.zip](https://github.com/user-attachments/files/18292755/crash_dump_00000003.zip)

**Comment by thepokemonsters** [comment]
im having the same problem. cant boot any games

**Comment by kporceil** [comment]
same

**Comment by Aqua-moonn** [comment]
Also getting this error, and have found no solution anywhere (Google, reddit, gbatemp, and the Nintendo homebrew discord)

**Comment by Masamune3210** [comment]
So you are all having issues with Persona Q2 specifically and no other game or app?

**Comment by TakumaEN** [comment]
@Masamune3210 yes, as weird as it sounds it is the only game I have downloaded in my system that crashes and returns that error.

**Comment by urherenow** [comment]
Then it’s a bad download, or you’re simply missing the seed…

**Comment by TakumaEN** [comment]
@urherenow Whoa, I wasn't familiarized with the whole seed thingjust looked a bit into seed, but I wan to try this hypothesis out. Sorry to ask such noob questions, but could you elucidate me on how do I find the seed and where should I place it in the files?

**Comment by urherenow** [comment]
Sorry, but no. Piracy is not supported in any way.

**Comment by TakumaEN** [comment]
@urherenow oh, it's one of those methods. I see, I see. Well, I'll try looking into it and return with the results. Might take me a while, but thanks, that gives me hope!

**Comment by Xancletaman** [comment]
you can fix the problem doing this; get inside fbi, look for the game with the probleam, an choose import seed
